src/megameklab/com/printing/InventoryWriter.java

```diff
@@ -486,9 +486,9 @@ private int calcLineCount(float fontSize, double viewWidth) {
             }
             lines += rows;
         }
-        lines += sheet.getTextLength(ammoText, fontSize) / viewWidth;
-        lines += sheet.getTextLength(featuresText, fontSize) / viewWidth;
-        lines += sheet.getTextLength(quirksText, fontSize) / viewWidth;
+        lines += Math.ceil(sheet.getTextLength(ammoText, fontSize) / viewWidth);
+        lines += Math.ceil(sheet.getTextLength(featuresText, fontSize) / viewWidth);
+        lines += Math.ceil(sheet.getTextLength(quirksText, fontSize) / viewWidth);
```

Comment:
While testing the aero record sheet branch I discovered that the font scaling was only counting full lines.